charts/pocket/values.yaml

```diff
@@ -100,6 +100,7 @@ config:
     use_rain_tree: true
     is_empty_connection_type: false
     private_key: "" # @ignored This value is needed but ignored - use privateKeySecretKeyRef instead
+    # TODO: I think this has been renamed to `max_nonces`
```

Comment:
Cna you please update it everywhere else as well?

![Screenshot 2023-07-13 at 4 14 38 PM](https://github.com/pokt-network/pocket/assets/1892194/3a11d1af-a1e6-4840-9430-40074b20133d)
![Screenshot 2023-07-13 at 4 14 31 PM](https://github.com/pokt-network/pocket/assets/1892194/a09297a8-6eb1-479f-aba7-06b3863e2d2c)
![Screenshot 2023-07-13 at 4 14 23 PM](https://github.com/pokt-network/pocket/assets/1892194/4e488a03-895f-4793-9f00-5e259cf4d007)